knowledge-base/how-to-upgrade-an-old-project-to-the-latest-telerik-reporting-version.md

```diff
@@ -0,0 +1,37 @@
+---
+title: How to upgrade an old project to the latest Telerik Reporting version

```

Comment:
You can change this according to the style guide - https://testdocs.telerik.com/style-guide/titles-and-headings#knowledge-base-articles . E.g. "Upgrading old projects to the latest Telerik Reporting version"

Reply:
Done